components/CreateReplyRequestForm/CreateReplyRequestForm.js

```diff
@@ -0,0 +1,238 @@
+      {disabled ? '字數太少，無法送出' : '送出理由'}
+    </button>
+  );
+};
+
+const CreateReplyRequestForm = React.memo(
+  ({ articleId, requestedForReply, onNewReplyButtonClick }) => {
+    const [disabled, setDisabled] = useState(false);
+    const [showForm, setShowForm] = useState(false);
+    const [text, setText] = useState('');
+
+    const [shareAnchor, setShareAnchor] = useState(null);
+
+    useEffect(() => {
+      // restore from localStorage if applicable.
+      // We don't do this in constructor to avoid server/client render mismatch.
+      //
+      setText(localStorage.text || text);
+    }, []);
+
+    const handleTextChange = ({ target: { value } }) => {
+      setText(value);
+      setDisabled(!value || value.length < MIN_REASON_LENGTH);
+      // Backup to localStorage
+      requestAnimationFrame(() => (localStorage.text = value));
+    };
+
+    const handleReasonSubmitted = () => {
+      setText('');
+      setDisabled(false);
+      requestAnimationFrame(() => (localStorage.text = ''));
+    };
+
+    const classes = useStyles();
+    const user = useCurrentUser();
+
+    return (
+      <div>
+        {showForm && (
+          <>
+            <Box pl={7} pt={2}>
+              <Hint>{t`Did you find anything suspicious about the message?`}</Hint>
+            </Box>
+            <Box component="form" display="flex" pt={2}>
+              <Box pr={2} pt={0.5}>
+                <Avatar user={user} size={42} />
+              </Box>
+              <Box flex={1}>
+                <div className={classes.form}>
+                  <textarea
+                    className={classes.textarea}
+                    placeholder="例：我用 OO 關鍵字查詢 Facebook，發現⋯⋯ / 我在 XX 官網上找到不一樣的說法如下⋯⋯"
+                    onChange={handleTextChange}
+                    value={text}
+                    rows={1}
+                  />
+                  <SubmitButton
+                    className={cx(classes.submit, classes.button)}
+                    articleId={articleId}
+                    text={text}
+                    disabled={disabled}
+                    onFinish={handleReasonSubmitted}
+                  />
+                </div>
+              </Box>
+            </Box>
+          </>
+        )}
+        <Box display="flex" py={2} alignItems="center">
+          <button
+            type="button"
+            className={cx(classes.button, classes.replyButton)}
+            onClick={onNewReplyButtonClick}
+          >
+            {t`Reply to this message`}
+          </button>
+          <div className={classes.buttonGroup}>
+            <button
+              type="button"
+              className={cx(showForm && 'active')}
+              onClick={() => setShowForm(!showForm)}
+            >{t`I second that`}</button>
+            <button
+              type="button"
+              className={cx(requestedForReply && 'active')}
+            >{t`I also wanna know`}</button>
```

Comment:
"I second that" button toggles the form that submits `replyRequest`, thus I think the state `requestedForReply` should link to that button. ( @LucienLee , do we have different state for "I second that" button if the user has requested reply? )

I don't see "I also wanna know" button in mockup now